azkaban-common/src/main/java/azkaban/database/DataSourceUtils.java

```diff
@@ -229,6 +235,11 @@ public boolean allowsOnDuplicateKey() {
     public String getDBType() {
       return "h2";
     }
+
+    @Override
+    public java.util.logging.Logger getParentLogger() throws SQLFeatureNotSupportedException {
```

Comment:
This method could be implemented inside azkabanDataSource, rather than implementing twice here.